### ISSUE TITLE
v2.0.0 clock.rb location

### ISSUE THREAD

**Issue by Epigene** [tomykaira/clockwork#183]
It seems that the executable is finicky about where the clock.rb file is.

I dislike keeping random *.rb files in app root, so I though placing it in `/config/initializers/clock.rb` would be cleaner, but calling `clockwork config/initializers/clock.rb` throws `cannot load such file -- ./config/initializers/clock.rb`

Similarly if I place `clock.rb` contents in `/.clock.rb`, calling `clockwork .clock.rb` fails.

Wouldn't it be awesome to either have clockwork accept file paths correctly or have `clockwork` call just work if one has /clock.rb placed correctly?

**Comment by paulgoetze** [comment]
I agree on having the clock.rb in the app’s root directory should not told to be a best practice. 
I put the clock.rb into `config/clock.rb`. When running 

```bash
$ bundle exec clockwork config/clock.rb
```
everything works fine for me with v2.0.0.
